src/api/app/jobs/report_to_scm_job.rb

```diff
@@ -25,14 +25,16 @@ def perform_job?(event_id)
     @event_type = @event.eventtype
     return false unless ALLOWED_EVENTS.include?(@event_type)
 
-    @event_package = Package.find_by_project_and_name(@event.payload['project'], Package.striping_multibuild_suffix(@event.payload['package']))
-    return false if @event_package.blank?
+    @event_package = Package.find_by_project_and_name(@event.payload['project'], Package.striping_multibuild_suffix(@event.payload['package'])) unless @event_type == 'Event::RequestStatechange'
+    @event_request = BsRequest.find_by_number(@event.payload['number'])
```

Comment:
I find this easier on the eyes...

```suggestion
    if @event_type == 'Event::RequestStatechange'
      @event_request = BsRequest.find_by_number(@event.payload['number'])
    else
      @event_package = Package.find_by_project_and_name(@event.payload['project'], 
      Package.striping_multibuild_suffix(@event.payload['package']))
    end
```

Reply:
done